Mobile/ShopLens/Droid/Activities/TextVoicerActivity.cs

```diff
@@ -30,7 +30,7 @@ public void OnInit([GeneratedEnum] OperationResult status)
             }
         }
 
-        private void SpeakOut()
```

Comment:
Like MJay asked, did you need these methods to not be private for some reason?

They're only relevant to this activity, so I don't see why they shouldn't be private

Reply:
"private modifier is redundant"
By default these are private